crawl-ref/source/dat/descript/monsters.txt

```diff
@@ -785,7 +785,7 @@ skilled in archery.
 centaur warrior
 
 A muscular centaur, veteran of numerous battles, armed with well-made weapons
-and able to use them skillfully.
+and able to use them skilful.
```

Comment:
```suggestion
and able to use them skilfully.
```

Still want the adverb

Reply:
Sorry; I thought I'd changed that. The "draconian knight" description also has the wrong spelling of "skilfully".